README.rst

```diff
@@ -77,19 +50,57 @@ which you need to build locally. To build default sample (`CDN`_) run::
+which Intel GPUs and media codec features will be supported. Some media stacks might
+also require special host setup instruction. See table below.
+
++--------------------------------------------+----------------------------------------+------------------------------------------------+----------------------------------+
+| Dockerfile                                 | Intel media stack origin               | Supported Intel GPUs                           | Host setup instructions          |
++============================================+========================================+================================================+==================================+
+| `docker/ubuntu20.04/native/Dockerfile`_    | Ubuntu 20.04                           | Gen8+, check Ubuntu 20.04 documentation        | Use any Linux distribution which |
+|                                            |                                        |                                                | supports required platform       |
++--------------------------------------------+----------------------------------------+------------------------------------------------+----------------------------------+
+| `docker/ubuntu20.04/intel-gfx/Dockerfile`_ | `Intel Graphics Package Repository`_   | `Intel® Iris® Xe MAX Graphics`_ (formerly DG1) | `DG1 Setup Instruction`_         |
++                                            +                                        +------------------------------------------------+----------------------------------+
+|                                            |                                        | Platforms starting from Gen8 and prior to DG1  | Use any Linux distribution which |
+|                                            |                                        |                                                | supports required platform       |
++--------------------------------------------+----------------------------------------+------------------------------------------------+----------------------------------+
+| `docker/ubuntu20.04/intel-gfx/Dockerfile`_ | Custom, specified via                  | Depends on the custom stack. Contact supplier.                                    |
+|                                            | ``--build-arg INTEL_GFX_APT_REPO=...`` |                                                                                   |
++--------------------------------------------+----------------------------------------+------------------------------------------------+----------------------------------+
+
+**Caveat:** current master branch is in active development to support DG2 and ATS-M GPUs
+via custom media stack (contact us for more details). Mind that latest release at
+`Intel Graphics Package Repository`_ is known to contain some outdated packages (for
+example, libvpl) which might have bugs affecting our sample command lines modified for
```

Comment:
bugs -> issues
(makes it less harsh on Intel)

Reply:
fixed